crates/oxc_linter/src/utils/unicorn.rs

```diff
@@ -0,0 +1,14 @@
+use oxc_ast::ast::Expression;
+
+pub fn is_dom_node_call(expr: &Expression) -> bool {
```

Comment:
This function name does not match the logic of this function, shall we just inline this instead of creating a util?

Reply:
There is the same util in `eslint-plugin-unicorn`. This function is helpful in other rules. We should keep it.

Reply:
You can see https://github.com/sindresorhus/eslint-plugin-unicorn/blob/2f77a236777c89314d91988d350424127696fec7/rules/utils/is-node-value-not-dom-node.js#L17.

I change the function name to `is_node_value_not_dom_node`, keeping the consistency with `eslint-plugin-unicorn`

Reply:
In such cases we should keep the exact function name, i.e. isNodeValueNotDomNode and also copy over the comments.

Reply:
For confusing functions, we should leave some clues to future contributors on what a function does.